cmd/openshift/operator/kodata/openshift-monitoring/02-chains-monitoring.yaml

```diff
@@ -0,0 +1,68 @@
+---
+# Copyright 2024 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: openshift-chains-read
+  namespace: tekton-chains
```

Comment:
@sudhishmk Do we deploy chains on `tekton-chains` namespace.
I hope we do transform the namespace before applying.
however it is good to have the default namespace of OpenShift environment
that is `openshift-pipelines`.

@concaf @piyush-garg please confirm the namespace.

Reply:
Yes we deploy chains in `openshift-pipelines` namespace in Openshift environment

Reply:
Thanks, will check that.

Reply:
Updated the namespace.